src/util/VehicleSpawner.cpp

```diff
@@ -2,36 +2,25 @@
 
 namespace YimMenu
 {
-	extern bool SpawnVehicle(std::string model_name, int player_ped_id)
+	int SpawnVehicle(std::string model_name, Vector3 coords, float rotation)
```

Comment:
Rotation in rage engine is associated with cartesian values of yaw, pitch and roll. This is more likely 'Heading'.

Reply:
Would you like me to change before merge, or I can just change it in my next pr.

Reply:
This can be changed another time. Not a pressing issue.

Reply:
> This can be changed another time. Not a pressing issue. I will merge this after testing the dll myself.

Okay, thanks

Reply:
Caging often results in the cage being held too high. This in turn makes for poor jailing and victims escape quite easily.

Reply:
It is an issue with the hat on the player i believe

Reply:
> Caging often results in the cage being held too high. This in turn makes for poor jailing and victims escape quite easily.

> It is an issue with the hat on the player i believe

In that case, isnt there a way to force the player to lose the hat?  Or getting the peds coords and setting the object coords to - 5 to ensure the cage lands on them and not above them?

Reply:
> > Caging often results in the cage being held too high. This in turn makes for poor jailing and victims escape quite easily.
> 
> 
> 
> > It is an issue with the hat on the player i believe
> 
> 
> 
> In that case, isnt there a way to force the player to lose the hat?  Or getting the peds coords and setting the object coords to - 5 to ensure the cage lands on them and not above them?

I am gonna play around with disabling the collisions on the cage later. I don't think there's a way to get the hat off. Theoretically, we can use SHOOT_SINGLE_SHOT_AT_COORD to get the hat off. that's my only idea and I will mess around with it tonight.

Reply:
Disabling collision for the cage may cause it to not even cage the player.  I wonder though, if we can disable the collision on the hat instead??

Reply:
> Disabling collision for the cage may cause it to not even cage the player.  I wonder though, if we can disable the collision on the hat instead??

Can you even get the player hat object?

Reply:
> > Disabling collision for the cage may cause it to not even cage the player.  I wonder though, if we can disable the collision on the hat instead??
> 
> Can you even get the player hat object?

Im assuming so, its a pickup for sure and it is always labeled "Hat" no matter what kind of hat it is, so theres gotta be something for it.

Reply:
Are you guys suggesting the Hats on player heads are causing the cage to be put too high? Surely, even if this is the case, this can be very easily resolved. Even a centered bone position on the ped should be enough to spawn a cage at the proper height.

Reply:
> Are you guys suggesting the Hats on player heads are causing the cage to be put too high? Surely, even if this is the case, this can be very easily resolved. Even a centered bone position on the ped should be enough to spawn a cage at the proper height.

@Deadlineem may have found a solution using the z value to fix it. I will try to implement the change later and I will let you know if it works. According to ur suggestion, how would you suggest to adjust this said bone?

Reply:
Look at the esp code, there should be bone Id's that would suit your current needs.

Reply:
> Look at the esp code, there should be bone Id's that would suit your current needs.

Okay I will take a look